src/App.tsx

```diff
@@ -1,27 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import 'bulma';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import { get5First, getAll, getRedGoods } from './api/goods';
+import { Select } from './types/Select';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[] | null>(null);
+  const [dataError, setDataError] = useState(false);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const getData = async (method:Select) => {
+    let data;
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+    try {
+      switch (method) {
+        case Select.all:
+          data = await getAll();
+          break;
+        case Select.firstFive:
+          data = await get5First();
+          break;
+        case Select.onlyRed:
+          data = await getRedGoods();
+          break;
+        default:
+          data = null;
+      }
```

Comment:
good job!
you can also pass needed function to make api call directly in function arguments instead of separate `method` variable